GeneralsMD/Code/GameEngine/Source/GameLogic/AI/AISkirmishPlayer.cpp

```diff
@@ -681,12 +681,16 @@ void AISkirmishPlayer::buildAIBaseDefenseStructure(const AsciiString &thingName,
 		Real s = sin(angle);
 		Real c = cos(angle);
 
-	  DEBUG_LOG(("buildAIBaseDefenseStructure -- Angle is %f sin %f, cos %f \n", 180*angle/PI, s, c));
+#if defined(RTS_DEBUG) || defined(RTS_INTERNAL)
+		// TheSuperHackers @info helmutbuhler 21/04/2025
+		// This log causes mismatch on Release when logging is enabled.
```

Comment:
These multiline comments could be simplified to `// TheSuperHackers @info Log causes mismatch in Release` for compactness.